Source/WebGPU/WebGPU/ComputePassEncoder.mm

```diff
@@ -71,11 +71,15 @@
 
 void ComputePassEncoder::dispatch(uint32_t x, uint32_t y, uint32_t z)
 {
+    m_device->getQueue().waitForAndClearPendingFences(m_computeCommandEncoder);
```

Comment:
I think this is wrong.

Reply:
Indeed :) I was missing a call to useResource, which is why the hazard tracking wasn't working